app/presenters/publishing_api_presenters/edition.rb

```diff
@@ -73,5 +81,20 @@ def default_update_type
     def specialist_sectors
       [edition.primary_specialist_sector_tag].compact + edition.secondary_specialist_sector_tags
     end
+
+    def access_limited
+      {
+        users: users.map(&:id)
+      }
+    end
+
+    def users
+      return @users if @users
+
+      org_users = edition.organisations.map(&:users).flatten
```

Comment:
It's a bit more efficient to do `User.where(organisation: edition.organisations)`.